### PR TITLE
nullness...

### PR DESCRIPTION
**Pull request by maggu2810**

### CODE REVIEW

File: bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/internal/WebClientFactoryImpl.java [hunk 1]
```diff
@@ -185,9 +188,9 @@ private int getConfigParameter(Map<String, Object> parameters, String parameter,
     private synchronized void initialize() {
         if (threadPool == null || commonHttpClient == null || commonWebSocketClient == null) {
             try {
-                AccessController.doPrivileged(new PrivilegedExceptionAction<Void>() {
+                AccessController.doPrivileged(new PrivilegedExceptionAction<@Nullable Void>() {
```
Comment (kaikreuzer) [inline bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/internal/WebClientFactoryImpl.java:191]: that's the trick I was missing...

**Comment by kaikreuzer** [comment]
Seems to be based on an older version...?

**Comment by maggu2810** [comment]
> Seems to be based on an older version...?

Perhaps, don't know how often you rebase your branch :wink:

**Comment by kaikreuzer** [comment]
Not a single time :-)

**Comment by maggu2810** [comment]
Holy sh***
It seems I can write to your repo and pushed the latest changes to it...

**Comment by maggu2810** [comment]
I edit the file twice.

* first edit
* pushed first edit to my repo
* second edit
* pushed second edit to your repo
* created PR from mine to yours

So, you already contains the latest changes of mine and this PR tries to set only a subset.
Sorry about that. Didn't want to push to your branch.

Your branch contains now my latest changes, too.
Do whatever you want with the latest commit of mine in your branch.

**Comment by kaikreuzer** [comment]
> It seems I can write to your repo and pushed the latest changes to it...

No worries, that is completely normal. You might have already seen the little checkbox when you create new PRs, which is checked by default and allows repo maintainers to do changes to the PR branch (but only this).

**Pull request closed without merging**